includes/admin/class-wc-admin-importers.php

```diff
@@ -227,16 +227,21 @@ public function do_ajax_product_import() {
 		// Log failures.
 		if ( 0 !== $params['start_pos'] ) {
 			$error_log = array_filter( (array) get_user_option( 'product_import_error_log' ) );
+			$import_log = array_filter( (array) get_user_option( 'product_import_log' ) );
 		} else {
 			$error_log = array();
+			$import_log = array();
 		}
 
 		$importer         = WC_Product_CSV_Importer_Controller::get_importer( $file, $params );
 		$results          = $importer->import();
 		$percent_complete = $importer->get_percent_complete();
 		$error_log        = array_merge( $error_log, $results['failed'], $results['skipped'] );
+		$import_log['imported']        = array_merge( $import_log['imported'], $results['imported'] );
+		$import_log['updated']        = array_merge( $import_log['updated'], $results['updated']);
 
 		update_user_option( get_current_user_id(), 'product_import_error_log', $error_log );
+		update_user_option( get_current_user_id(), 'product_import_log', $import_log );
```

Comment:
It get update, but there's no real use on WooCommerce for it.

Reply:
I can see the use for this hooks could be in many ways like:
- importing product details from third party source  
- import media from other online resource 
- apply dynamic changes during the import export 

so as the importing process is going, you have the import log to apply the dynamic changes or make API Call for further data during the Ajax call.

main use would be for plugins like multi vendor , multi language or auto listing solutions. usesare not limited for what i mentioned.

extending default woocommerce importer is important to provide integration between multiple advanced plugins which now each had created their own solutions.